packages/actions/docs/07-prebuilt-actions/08-import.md

```diff
@@ -495,6 +495,17 @@ ImportAction::make()
 
 If you are encountering memory issues when importing large CSV files, you may wish to reduce the chunk size.
 
+## Changing the field delimiter
+
+The default delimiter for CSV is the comma (`,`).
+If your import will use a different delimiter, you may call the `delimiter()` method on the action. This also works in conjunction with `example()` when writing a file. NOTE: You must specify only a single character, else an exception will be thrown:
+
+```php
+ImportAction::make()
+    ->importer(ProductImporter::class)
+    ->delimiter(';')
+```
```

Comment:
```suggestion
The default delimiter for CSVs is the comma (`,`).
If your import should use a different delimiter, you may call the `delimiter()` method on the action, passing a new one:

```php
ImportAction::make()
    ->importer(ProductImporter::class)
    ->delimiter(';')
```

You can only specify a single character, otherwise an exception will be thrown.
```